setup.cfg

```diff
@@ -37,11 +37,14 @@ install_requires =
 
     # default numpy requirements
     numpy==1.13.3; python_version=='3.5' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX'
-    numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX'
+    numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
+    numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
     numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64'
```

Comment:
Should this have ` and platform_python_implementation != 'PyPy'` I know PyPy doesn't have 3.8 but when it comes out it will likely need at least 1.19 so it might make sense to declare this even if it's not used now and won't be for awhile. It may make sense to add it below but maybe that pinning happens when it's actually released and at it will be unpinned instead of pinned to 1.17.3 which might be not the right version to use.

Reply:
I'm not sure if I stand by this comment since numpy 1.17.3 built from source would probably be fine, but not sure if this package prefers the min supported version or the min supported wheel for that version.